tools/lib/lolo_log_puller/lolo_log_puller_cli.py

```diff
@@ -97,31 +97,27 @@ def main():
         app_unit_clean_name = app_unit.replace('/', '-')
         # Note: /var/lib/charm may not always exist (ceph stores confs there)
 
-        cmd_archive = '''
```

Comment:
Splitting this huge command into small ones makes it clear on failure which one of all these commands has actually failed.